spec/features/providers/allocations/show_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+# Form is currently not yet implemented
+
+feature "Show providers", type: :feature do
```

Comment:
💡 Prepend `x` to `feature`  so that rspec output indicates that the spec is pending

Reply:
I've now deleted this file since it does nothing